build/config/build.yaml

```diff
@@ -82,4 +87,61 @@ steps:
   dir: 'tests/'
   waitFor: ['DOCKER_STAGE', 'DOCKER_BUILD_MVN_GCLOUD']
 
+#
+# End of App Engine Integration tests
+
+#
+# GKE Integration tests
+#
+
+# Create a Docker image with the test application
+- name: 'gcr.io/cloud-builders/docker'
+  id: 'DOCKER_BUILD_TEST_APPLICATION'
+  args: ['build', '--tag=${_STAGING_IMAGE}-test-app', '--no-cache', 'tests/runtimes-common-testing/target/deploy']
```

Comment:
You can just exclude the module from being built/cleaned by JETTY_SMOKE_TEST by using `-pl !<module_name>`